spec/prog/postgres/postgres_nexus_spec.rb

```diff
@@ -172,6 +172,58 @@
     end
   end
 
+  describe "#refresh_server_certificate" do
+    let(:postgres_server) {
+      PostgresServer.create_with_id(
+        project_id: SecureRandom.uuid,
```

Comment:
nit: You might want to use a hardcoded value https://github.com/ubicloud/ubicloud/pull/203#discussion_r1228280995

Also it might be good to add foreign_key to the project_id column. It will prevent setting a non-existent project ID.